js_modules/dagit/packages/core/src/partitions/PartitionStatus.tsx

```diff
@@ -90,6 +90,8 @@ export const PartitionStatus: React.FC<{
     [partitionNames],
   );
 
+  const selectedSet = React.useMemo(() => new Set(selected), [selected]);
```

Comment:
Worth doing a `JSON.stringify()` on `selected` (maybe `JSON.stringify([...selected].sort())` or something) in case it's a new object on every render?